components/image/image.css

```diff
@@ -0,0 +1,31 @@
+
+:root{
+    --grey:grey;
+    --white-light:rgb(248, 244, 244);
+}
+.img-parent{
+    width:31.25rem;
+    border:1px solid var(--grey);
+    padding:1.56rem;
+    width:fit-content;
+    background-color: var(--white-light);
+}
+.img-parent img{
```

Comment:
If possible try avoiding css chaining, in this case i guess by using only .img-parent also the work can be done